AdyenActions/Components/Await/PollingHandlerProvider.swift

```diff
@@ -34,12 +34,15 @@ internal struct PollingHandlerProvider: AnyPollingHandlerProvider {
     private let apiContext: APIContext
 
     /// :nodoc:
-    private let apiClient: AnyRetryAPIClient?
+    private let apiClient: AnyRetryAPIClient
 
     /// :nodoc:
-    internal init(apiContext: APIContext, apiClient: AnyRetryAPIClient?) {
```

Comment:
this wasn't used in unit tests to inject `AnyRetryAPIClient `?

Reply:
nope, did not find any use in tests